### DISCUSSION TITLE
Select the automatic mode sometimes the rate remains stable but after a while it becomes unstable

### DISCUSSION THREAD

**Discussion by 15976770947** [Mygod/VPNHotspot#310] [Q&A]
When I select the automatic mode, sometimes the rate remains stable, but it becomes unstable after a while. What's the matter? I tried it on other devices. How can I solve it?

**Comment by 15976770947** [comment]
I also found that choosing one of the 5g channels will automatically turn off after a few hours after using it. You need to turn on the wireless relay again. I have not found this situation in 2.4g.

**Comment by 15976770947** [comment]
I also found that in the Android 9 version of the system, the WiFi hotspot is turned on, but the icon is turned off. It is actually turned on. After opening the sharing vpn button, youtube and other websites cannot be opened, but access to the google search website is normal.

**Reply by Mygod** [reply]
Disable hardware acceleration.

**Comment by Mygod** [comment]
As for repeater, it sounds like an issue of your hardware/ROM not properly handling Wi-Fi direct. You might be able to mitigate this by preventing your device from connecting to other Wi-Fi APs.
